src/Drawing.js

```diff
@@ -21,11 +21,13 @@ const Point = require("./Point");
 const Spline = require("./Spline");
 const Ellipse = require("./Ellipse");
 const TagsManager = require("./TagsManager");
+const TagsStreamManager = require("./TagsStreamManager");
 const Handle = require("./Handle");
 const Mesh = require("./Mesh");
 
 class Drawing {
-    constructor() {
+    constructor(stream) {
```

Comment:
I'm confused as a stream is passed in as a constructor option, but `this.stream` is never used.  Can the constructor argument be dropped?

Reply:
Yes my bad this is not needed.